src/v/config/configuration.cc

```diff
@@ -273,6 +273,12 @@ configuration::configuration()
       "wasn't reached",
       required::no,
       1ms)
+  , alter_topic_cfg_timeout_ms(
+      *this,
+      "alter_topic_cfg_timeout_ms",
+      "Time to wait for alter topic configuration",
```

Comment:
why is there a debounce timeout, is this so that we batch configurations?

would it be simpler if we didn't have that?

Reply:
we need to have timeout to limit the time that we wait for replicate to finish, when we call replicate in `alter_configs` request.

Reply:
so the description doesn't say anything about waiting to replicate the contents of alter_config when coming from the kafka layer. So let's add that context. 

i don't know, generically, what it means to `wait for alter topic configuration` what are we `actually waiting for` what part of the wait. is it replicate or saving to disk, or whatev. so the description should be useful to a non expert.